cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -169,6 +182,93 @@ protected void setMetricsEnabled(boolean isMetricsEnabled) {
         this.isMetricsEnabled = isMetricsEnabled;
     }
 
+    protected String getLogConfig() {
+        return logConfig;
+    }
+
+    private Map getDefaultLogConfig() {
+        Map<String, String> defaultSettings = new LinkedHashMap<>();
+        // default values
+        defaultSettings.put("log4j.appender.CONSOLE", "org.apache.log4j.ConsoleAppender");
```

Comment:
You are basically mirroring the fields from the `log4j.properties` files in the Docker images and if the user doesn't configure it, you will generate the file out of these defaults. Do we still need the `log4j.properties` files? Or can we just delete them from the Docker images?

Reply:
We can delete them.